### PR TITLE
fix build error for newer JDK version

### PR DESCRIPTION
**Pull request by gautamjajoo**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,9 +33,10 @@ dependencies {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
     manifest {
         attributes(
-                'Class-Path': configurations.compile.collect { it.getName() }.join(' '),
+                'Class-Path': configurations.runtimeClasspath.collect { it.getName() }.join(' '),
```
Comment (vincenzopalazzo) [inline build.gradle:39]: there is any resource where tell that compiles equal to runtimeCallpath?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (vincenzopalazzo) [reply build.gradle:39]: https://github.com/linkedin/cruise-control/issues/1699#issuecomment-923890453

Comment (gautamjajoo) [reply build.gradle:39]: Yes this was the fix

Comment (vincenzopalazzo) [reply build.gradle:39]: maybe yes, you need to check it in the local machine if you want support JDK 18

**Comment by vincenzopalazzo** [comment]
Included with the `compileClasspath` here https://github.com/clightning4j/jrest/pull/30

Thanks for report this out

**Pull request closed without merging**